replication_impl.go

```diff
@@ -25,8 +25,55 @@ package driver
 import (
 	"context"
 	"path"
+	"strconv"
 )
 
+type batch_params struct {
+	ttl float64 `json:"ttl"`
+}
+
+// CreateBatch creates a "batch" to prevent WAL file removal and to take a snapshot
+func (c *client) CreateBatch(ctx context.Context, serverID int64, db Database) (BatchMetadata, error) {
+	req, err := c.conn.NewRequest("POST", path.Join("_db", db.Name(), "_api/replication/batch"))
+	if err != nil {
+		return BatchMetadata{}, WithStack(err)
+	}
+	req = req.SetQuery("serverId", strconv.FormatInt(serverID, 10))
+	params := batch_params{ttl: 60.0} // just use a default ttl value
```

Comment:
First: The default TTL should be specified in a `const`.
Second: I suggest to make the TTL configurable with a context function `WithBatchTTL`